src/Lesson5/ArrayThreads.java

```diff
@@ -0,0 +1,118 @@
+package Lesson5;
+
+import java.util.ArrayList;
+
+public class ArrayThreads {
+    static final int size = 10000000;
+    static float[] arr = new float[size];
+    static int numParts = 3; // Задаем количество частей, на которые будет разбит исходный массив
```

Comment:
```suggestion
    static final int NUM_PARTS = 3; // Задаем количество частей, на которые будет разбит исходный массив
```
поскольку это является константой